docs: useUserShipping docs update (#361)
vuestorefront/bigcommerce: docs/composables/use-user-shipping.md

## Features

`useUserShipping` composable is used to:

- fetch saved customer addresses,
- submit new shipping addresses,
- modify and delete existing shipping addresses.

## API

- `addAddress` - function for adding new customer shipping address and populating the `shipping` object with the updated result. This method accepts a single object parameter. See [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.

- `deleteAddress` - function for deleting existing shipping address and populating the `shipping` object with the updated result. This method accepts a single object parameter. See [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.

- `updateAddress` - function for updating existing shipping address and populating the `shipping` object with the updated result.. This method accepts a single object parameter with following properties:

    - `address: UserShippingAddressItem` - See [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.

>     - `shipping: UserShippingAddressItem[]` - See - See [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.

Reviewer: ```suggestion
    - `shipping: UserShippingAddressItem[]` - See the [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.
```

> - `load` - function for getting shipping information from the BigCommerce plaform. This method populates the `shipping` object with the result.

Reviewer: ```suggestion
- `load` - a function for getting shipping information from the API This method populates the `shipping` object with the returned data.
```

> - `shipping: UserAddress[]` - a main data object. See [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.

Reviewer: ```suggestion
- `shipping: UserAddress[]` - a main data object. See the [UserShippingAddressItem](../reference/api/bigcommerce-api.usershippingaddressitem.md) type alias for more information.
```

> - `loading: boolean` - a reactive object containing information about the loading state of the shipping information.

Reviewer: ```suggestion
- `loading: boolean` - a reactive object containing information about the loading state of the methods described above.
```

> - `error: UseUserShippingErrors` - reactive object containing the error message if any of the methods above failed for any reason. See the [UseUserShippingErrors](https://docs.vuestorefront.io/v2/reference/api/core.useusershippingerrors.html) interface for more information.

Reviewer: ```suggestion
- `error: UseUserShippingErrors` - a reactive object containing the error message if any of the methods above failed for any reason. See the [UseUserShippingErrors](https://docs.vuestorefront.io/v2/reference/api/core.useusershippingerrors.html) interface for more information.
```
